draft-ietf-quic-transport.md

```diff
@@ -1065,6 +1066,12 @@ sending a packet with a number of 0x6b2d79 requires a packet number encoding
 with 14 bits or more; whereas the 30-bit packet number encoding is needed to
 send a packet with a number of 0x6bc107.
 
+An endpoint MUST discard packets with duplicate packet numbers.  That is, a
+packet is discarded if the reconstructed packet number is equal to the
+reconstructed packet number of a previously received and processed packet.  Note
+however that endpoints MUST attempt to remove packet protection before
+discarding packets (see Section 9.3 of {{QUIC-TLS}}).
```

Comment:
I think i would put this in the TLS document S 5.4 instead. That way it's in logical order and this requirement seems like less of an aside.

Perhaps:

```
After unprotecting a packet, the receiver MUST check that it has not already processed
a packet with this PN and if so discard it. Duplicate suppression MUST happen after
the packet has been unprotected for the reasons described in Section 9.3. An efficient
algorithm for duplicate suppression can be found in RFC 2406 Section 3.4.3.```